### ISSUE TITLE
Using strings as axis labels does not work

### ISSUE THREAD

**Issue by SimonHeybrock** [scipp/scipp#741]
```---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-35-5153d03ec1a0> in <module>
      1 d.labels['xlabels'] = sc.Variable([Dim.X], values=['a','b','c','d','e'])
----> 2 plot(d['data_2d'][Dim.Y, 0], axes=['xlabels'])

~/software/scipp/scipp/plot/plot.py in plot(input_data, collapse, color, projection, axes, **kwargs)
     99                                    projection=projection,
    100                                    axes=val["axes"],
--> 101                                    **kwargs)
    102     return output

~/software/scipp/scipp/plot/dispatch.py in dispatch(input_data, ndim, name, collapse, sparse_dim, bins, projection, color, **kwargs)
     35                            color=color, **kwargs)
     36     elif projection == "1d":
---> 37         return plot_1d(input_data, color=color, **kwargs)
     38     elif projection == "2d":
     39         return plot_2d(input_data, name=name, **kwargs)

~/software/scipp/scipp/plot/plot_1d.py in plot_1d(input_data, logx, logy, logxy, color, filename, axes, mpl_axes)
     34 
     35     sv = Slicer1d(input_data=input_data, layout=layout, axes=axes, color=color,
---> 36                   mpl_axes=mpl_axes)
     37 
     38     if mpl_axes is None:

~/software/scipp/scipp/plot/plot_1d.py in __init__(self, input_data, layout, axes, color, mpl_axes)
     99             if self.slider[key].disabled:
    100                 button.disabled = True
--> 101         self.update_axes(str(list(self.slider_dims.keys())[-1]))
    102 
    103         self.ax.set_ylabel(ylab)

~/software/scipp/scipp/plot/plot_1d.py in update_axes(self, dim_str)
    168 
    169         new_x = self.slider_x[dim_str].values
--> 170         xc = edges_to_centers(new_x)
    171 
    172         for i, (name, var) in enumerate(sorted(self.input_data)):

~/software/scipp/scipp/plot/tools.py in edges_to_centers(x)
     24     Convert array edges to centers
     25     """
---> 26     return 0.5 * (x[1:] + x[:-1])
     27 
     28 

TypeError: __getitem__(): incompatible function arguments. The following argument types are supported:
    1. (self: scipp._scipp.core.VariableView_string, arg0: int) -> str

Invoked with: ["a", "b", ..., "d", "e"], slice(1, None, None)```

**Comment by nvaytet** [comment]
Can you post the code snippet that generated the above error? Thanks!

**Comment by SimonHeybrock** [comment]
```python
import numpy as np
import scipp as sc
from scipp import Dim
from  scipp.plot import plot

d = sc.DataArray(
    data=sc.Variable([Dim.X], values=np.arange(4)),
    coords={Dim.X: sc.Variable([Dim.X], values=['a','b','c','d'])})

plot(d)
```

**Comment by dtasev** [comment]
Closed by #783
